src/main/java/edu/greenblitz/tobyDetermined/subsystems/swerve/SwerveChassis.java

```diff
@@ -242,5 +242,13 @@ public enum Module {
 		BACK_LEFT,
 		BACK_RIGHT
 	}
-	
+
+
+	/** set the idle mode of the linear motor to brake */
+	public void setIdleModeBrake (Module module){
```

Comment:
Need to be set all modules to brake/coast

Reply:
done!